remoteit/desktop
# DESK-807 replace system confirm

### Description


replace system confirm dialog so that we don't lose focus in Windows

### Test plan



1. Sign in on Windows
2. Go to settings page
3. Click sign out
4. You should see the login page active and ready to enter data.

### Ticket



<https://remoteit.atlassian.net/browse/DESK-807>
